git-push-service/src/arrange.ts

```diff
@@ -24,7 +25,7 @@ export const arrangeManifests = async (inputs: Inputs): Promise<void> => {
 
 const arrangeNamespaceLevelManifests = async (inputs: Inputs): Promise<void> => {
```

Comment:
📝 This will be removed in https://github.com/quipper/monorepo-deploy-actions/pull/1201